src/paulscode/android/mupen64plusae/DrawerDrawable.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Mupen64PlusAE, an N64 emulator for the Android platform
+ * 
+ * Copyright (C) 2013 Paul Lamb
+ * 
+ * This file is part of Mupen64PlusAE.
+ * 
+ * Mupen64PlusAE is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * Mupen64PlusAE is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with Mupen64PlusAE. If
+ * not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Authors: littleguy77
+ */
+package paulscode.android.mupen64plusae;
+
+import android.graphics.drawable.Drawable;
+import android.graphics.Canvas;
+import android.graphics.PixelFormat;
+import android.graphics.ColorFilter;
+import android.graphics.Paint;
+
+public class DrawerDrawable extends Drawable
```

Comment:
Same as the previous background, except this one adds a white line along the right side. Should probably explain this in a comment instead...